tools/batchTestGenerator/README.md

```diff
@@ -51,4 +62,11 @@ Max number of jobs to include in `test-case-batch` file.
 --output=./ \
 --maxJobs=10 \
 --include=ECS,EKS
+```
+
+```
```

Comment:
Thank you so much for detailed PR description, can we possibly have an optional flag to include like `aocImage` to run test cases with custom `adot collector image`? or we can plan to use `common.tf` for these specific use-cases. :-)

Reply:
AOC Image is already customizable by setting the `TF_VAR_aoc_version` before running `make execute-batch-test`.

Reply:
Unless there is a use case where different tests will use different images then I do not believe it is in scope of this tool.

Reply:
yeah agreed